assets/src/edit-story/components/panels/stories/color.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import ColorPanel from '../color';
+
+export default {
+  title: 'Panels/Color',
+  component: ColorPanel,
+};
+
+export const _default = () => {
+  const color = text('Initial Color', '#4891fc');
+
```

Comment:
Don't we need
`const gradients = boolean('With Gradients', true);` here?

Reply:
Actually, I need to remove those stories, and also the "with gradients" part as I removed gradient support from the picker.